components/common/accordion/accordion.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import AccordionItem from './accordion_item';
+import './accordion.scss';
+
+export type AccordionData = {
+    title: string;
+    description: string;
+    items: React.ReactNode;
+};
+
+type Props = {
+    items: AccordionData[];
+    openMultiple?: boolean;
+    headerClick: <T>(...args: T[]) => T | void;
```

Comment:
The accordion will contain:
- items = will be all the possible accordions elements (li) to show.
- openMultiple = allow only one accordion open at a time, or multiple accordions open at a time.
- headerClick = a cb fn that will return a ref obj pointing to the accordion, so you can use to manipulate that from where you are using the common component.